src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -73,6 +73,8 @@ public static partial class PlatformDetection
         public static bool Is64BitProcess => IntPtr.Size == 8;
         public static bool IsNotWindows => !IsWindows;
 
+        public static bool IsPrivilegedProcess => IsUnixAndSuperUser || IsWindowsAndElevated;
+
```

Comment:
Do we still need the split across Unix/Windows, or can we view just have the implementation for this in the shared file and delete the OS specialization?

Reply:
Refactored per that suggestion; thanks.